Makefile

```diff
@@ -120,6 +120,23 @@ pkg/openapi/client.go: bin/openapi-generator-cli api/openapi/model-registry.yaml
 		--ignore-file-override ./.openapi-generator-ignore --additional-properties=isGoSubmodule=true,enumClassPrefix=true,useOneOfDiscriminatorLookup=true
 	gofmt -w pkg/openapi
 
+# Start the MySQL database
+.PHONY: start/mysql
+start/mysql:
+	./scripts/start_mysql_db.sh
+
+# Stop the MySQL database
+.PHONY: stop/mysql
+stop/mysql:
+	./scripts/teardown_mysql_db.sh
+
+# generate the gorm structs
+.PHONY: gen/gorm
+gen/gorm: bin/gorm-gen bin/golang-migrate start/mysql
```

Comment:
just to double-check:
- we intentionally leave this out from being a dependency from standard build target
- this intended to be run explicitly when the dev has made schema changes
- we "guard" this has been effectively run when we receive PR via the invocation on GHA.

if that is correct, I concur this be documented per @rareddy comment above

Reply:
> just to double-check:
> 
> * we intentionally leave this out from being a dependency from standard build target
> * this intended to be run explicitly when the dev has made schema changes
> * we "guard" this has been effectively run when we receive PR via the invocation on GHA.
> 
> if that is correct, I concur this be documented per @rareddy comment above

All of the above is correct, I'll add a section about it in the README.md or maybe it makes more sense to put that into CONTRIBUTING.md

Reply:
For the time being, I can add instructions here:  https://github.com/kubeflow/model-registry/blob/main/README.md#development